public/components/trace_analytics/requests/services_request_handler.ts

```diff
@@ -197,30 +196,6 @@ export const handleServiceMapRequest = async (
     }
   }
 
-  if (currService) {
-    await handleDslRequest(http, DSL, getRelatedServicesQuery(currService), mode, dataSourceMDSId)
-      .then((response) =>
-        response.aggregations.traces.buckets.filter((bucket: any) => bucket.service.doc_count > 0)
-      )
-      .then((filteredBuckets) => {
-        const maxNumServices = Object.keys(map).length;
-        const relatedServices = new Set<string>();
-
-        // Iterate through the filtered trace buckets
-        for (let i = 0; i < filteredBuckets.length; i++) {
-          // Add the current service name to the related services
-          relatedServices.add(filteredBuckets[i].key);
-          if (relatedServices.size === maxNumServices) break;
-        }
-
-        // Add the related services to the map for the current service
-        map[currService].relatedServices = [...relatedServices];
```

Comment:
nit. also remove https://github.com/opensearch-project/dashboards-observability/blob/8016a73671d6793c0217cf0442a8b3f6740bdd89/public/components/trace_analytics/components/common/plots/service_map.tsx#L49-L49
and the ones in https://github.com/opensearch-project/dashboards-observability/blob/8016a73671d6793c0217cf0442a8b3f6740bdd89/test/constants.ts#L446-L446?